### PR TITLE
Add study groups support

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
@gideonthomas R?

File: app/scienceapi/study_groups/models.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+class StudyGroup(models.Model):
+    """
+    Study groups on science site
+    """
+    name = models.CharField(max_length=300)
+    link = models.URLField(
+        max_length=500,
+        null=True,
```
Comment (gideonthomas) [inline app/scienceapi/study_groups/models.py]: not null or blank

File: app/scienceapi/study_groups/models.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    )
+    region = models.CharField(
+        max_length=300,
+        null=True,
+        blank=True,
+        help_text='Asia, North America, etc.'
+    )
+
+    class Meta:
+        get_latest_by = 'date_created'
```
Comment (gideonthomas) [inline app/scienceapi/study_groups/models.py]: remove

File: app/scienceapi/study_groups/models.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    def __str__(self):
+        return str(self.name)
```
Comment (gideonthomas) [inline app/scienceapi/study_groups/models.py]: `str('{name}, {location}, {region}'.format(name=self.name, location=self.location, region=self.region))`

**Comment by gideonthomas** [comment]
r+

**Pull request merged**